tensorflow/core/kernels/non_max_suppression_op_test.cc

```diff
@@ -861,4 +861,474 @@ TEST_F(NonMaxSuppressionWithOverlapsOpTest, TestEmptyInput) {
   test::ExpectTensorEqual<int>(expected, *GetOutput(0));
 }
 
+class CombinedNonMaxSuppressionOpTest : public OpsTestBase {
+ protected:
+  void MakeOp(bool pad_per_class = false) {
+    TF_EXPECT_OK(NodeDefBuilder("combined_non_max_suppression_op",
+                                "CombinedNonMaxSuppression")
+                     .Input(FakeInput(DT_FLOAT))
+                     .Input(FakeInput(DT_FLOAT))
+                     .Input(FakeInput(DT_INT32))
+                     .Input(FakeInput(DT_INT32))
+                     .Input(FakeInput(DT_FLOAT))
+                     .Input(FakeInput(DT_FLOAT))
+                     .Attr("pad_per_class", pad_per_class)
+                     .Finalize(node_def()));
+    TF_EXPECT_OK(InitOp());
```

Comment:
can we run non batched version to check the results.

Reply:
The input values and outputs I've chosen are same as the non batched version.